test/integration/integration_test.go

```diff
@@ -66,9 +66,23 @@ func TestMain(m *testing.M) {
 	}
 	runner := util.MinishiftRunner{CommandPath: minishiftBinary}
 	if runner.IsCDK() {
-		godogTags += "~minishift-only"
+		godogTags += "~minishift-only&&~b2d-only&&~centos-only"
 	} else {
-		godogTags += "~cdk-only"
+		switch os.Getenv("MINISHIFT_ISO_URL") {
+		case "b2d":
+			godogTags += "~cdk-only&&~centos-only&&~rhel-only"
```

Comment:
I am also getting a bit confused about how we are going with these tags/annotations. There is the CDK non CDK split and the centos-only now. Also why would we exclude 'centos-only' from the CDK tests?

Is the distinction between centos and rhel really needed?